packages/server/src/app.ts

```diff
@@ -205,3 +206,25 @@ export async function shutdownApp(): Promise<void> {
     rmSync(binaryStorage.replace('file:', ''), { recursive: true, force: true });
   }
 }
+
+const loggingMiddleware = (req: Request, res: Response, next: NextFunction):void=>{
+  const ctx = getRequestContext();
+  const start = new Date();
+  next();
+  const afterNext = new Date().getTime(); // Record the time after next() completes
+  const totalTime = afterNext - start.getTime(); // Calculate the time taken including the time spent in next()  
+  let userProfile: string | undefined;
+  if (ctx instanceof AuthenticatedRequestContext) {
+    userProfile = ctx.profile.reference;
+  }
+
+  ctx.logger.info('Request served', 
+  {
+    receivedAt:start,
+    requestMethodAndPath: `${req.method}: ${req.path}`,
```

Comment:
Could you split this into two different properties: `method` and `path`?  That will make it easier to query the logs for users